### PR TITLE
feat: :sparkles: (llm) new accounts and assets lists portfolio

### PR DESCRIPTION
**Pull request by LucasWerey**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - Wallet screen
  - Native UI

### 📝 Description

> [!IMPORTANT]  
> There is a test that's not working properly because of the animation. It looks like the shared values are not updated so the translation and the opacity change don't happen in test env. It has been decided to just check if the row exists

Integration of the new assets and accounts lists to the current portfolio screen. 
useListsAnimation.ts has the logic that ensure the animation of the tab switch is smooth.
Creation of a new component in the navite lib. I've renamed the current component to DrawerTabComponent since it's used only in a drawer and it looks more like a drawer tab selector.
Since the rename of the DrawerTabComponent is out of the scope I didn't change anything in it just a rename
Also reduce the margin top of the quick actions
Remove bottom edger on Accounts and Assets lists screens

Storybook link : https://native-ui-storybook-git-feat-live-14723-ledgerhq.vercel.app/?path=/docs/form-tabselector--docs

### ANIMATIONS DEMO

https://github.com/user-attachments/assets/5ef58a9b-c9c5-4e85-906b-b3c983cdb60e

https://github.com/user-attachments/assets/4f1cd06f-acf4-4332-b984-9b02ad02922b

### USE CASES DEMO

https://github.com/user-attachments/assets/ce11edcc-5d84-47b6-81eb-6e0cf77fe0de

### QUICK ACTIONS MARGIN 

image

### ❓ Context

- **JIRA or GitHub link**: [LIVE-14723](https://ledgerhq.atlassian.net/browse/LIVE-14723) 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-14724]: https://ledgerhq.atlassian.net/browse/LIVE-14724?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[LIVE-14723]: https://ledgerhq.atlassian.net/browse/LIVE-14723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/newArch/features/Accounts/components/AddAccountButton/index.tsx [hunk 1]
```diff
@@ -20,12 +20,16 @@ const StyledPressable = styled(Pressable)`
...
+  sourceScreenName: string;
+};
+
+const AddAccountButton: React.FC<Props> = ({ sourceScreenName }) => {
   const { t } = useTranslation();
   const [isAddModalOpened, setAddModalOpened] = useState<boolean>(false);
 
   const openAddModal = () => {
-    track("button_clicked", { button: "add a new account", page: "Accounts" });
+    track("button_clicked", { button: "Add a new account", page: sourceScreenName });
```
Comment (themooneer) [inline apps/ledger-live-mobile/src/newArch/features/Accounts/components/AddAccountButton/index.tsx:32]: nice catch, should it be under the FF ?

File: apps/ledger-live-mobile/src/screens/Portfolio/ListsSection.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+type ListsSectionProps = {
+  t: (key: string) => string;
+  handleToggle: (value: string) => void;
+  handleLayout: (event: LayoutChangeEvent) => void;
+  assetsAnimatedStyle: BaseAnimationStyle;
+  accountsAnimatedStyle: BaseAnimationStyle;
+  maxItemsToDysplay: number;
+};
+
+const ListsSection: React.FC<ListsSectionProps> = ({
```
Comment (themooneer) [inline apps/ledger-live-mobile/src/screens/Portfolio/ListsSection.tsx:31]: i suggest a `TabSection` name

File: libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+};
+
+interface TabSelectorProps<T extends string | number> {
+  options: T[];
+  selectedOption: T;
+  handleSelectOption: (option: T) => void;
+  labels: { [key in T]: string };
+}
+
+export default function DrawerTabSelector<T extends string | number>({
```
Comment (themooneer) [inline libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx:64]: why not `TabSelector` ? i mean in term of UI it's not a drawer , am i wrong ?

File: libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  selectedOption,
+  handleSelectOption,
+  labels,
+}: TabSelectorProps<T>): JSX.Element {
+  const { colors } = useTheme();
+
+  const longuestLabel =
+    labels[options[0]].length > labels[options[1]].length ? options[0] : options[1];
+
+  const widthFactor = 8;
```
Comment (themooneer) [inline libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx:75]: can be moved as a constant on top of the script or in a constants.ts script

File: libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  const margin = 20;
```
Comment (themooneer) [inline libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx:76]: `BASE_MARGIN` and can be moved as a constant on top of the script or in a constants.ts script

File: libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx (hunk 3 above)
Comment (LucasWerey) [reply libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx:64]: out of scope

File: libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+interface OptionButtonProps<T> {
+  option: T;
+  selectedOption: T;
+  handleSelectOption: (option: T) => void;
+  label: string;
+  width: number;
+}
+
+const OptionButton = <T,>({
```
[the remark this answers is not in the document]
Comment (LucasWerey) [reply libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx:37]: out of scope

File: libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx (hunk 5 above)
Comment (LucasWerey) [reply libs/ui/packages/native/src/components/Form/DrawerTabSelector/index.tsx:76]: out of scope

File: apps/ledger-live-mobile/src/newArch/features/Accounts/components/AddAccountButton/index.tsx (hunk 1 above)
Comment (LucasWerey) [reply apps/ledger-live-mobile/src/newArch/features/Accounts/components/AddAccountButton/index.tsx:32]: it's already the case it's only used inside the new AccountsList

File: apps/ledger-live-mobile/src/screens/Portfolio/ListsSection.tsx (hunk 2 above)
Comment (LucasWerey) [reply apps/ledger-live-mobile/src/screens/Portfolio/ListsSection.tsx]: Done

Comment (LucasWerey) [reply apps/ledger-live-mobile/src/screens/Portfolio/ListsSection.tsx]: Done ✅

**Review by themooneer — APPROVED** [review]
LGTM

**Pull request merged**